File: dynabuffers-java/src/main/kotlin/dynabuffers/ast/NamespaceType.kt
package dynabuffers.ast

import dynabuffers.api.ISerializable
import dynabuffers.api.IType

data class NamespaceType(val options: NamespaceTypeOptions, val nestedNamespaces : List<NamespaceType> = listOf()) : IType {

    data class NamespaceTypeOptions(val name: String, val list: List<ISerializable>)

}

File: dynabuffers-python/dynabuffers/ast/NamespaceType.py
from dynabuffers.api.ISerializable import ISerializable


class NamespaceTypeOptions:
    def __init__(self, name, list: [ISerializable]):
        self.name = name
        self.list = list


class NamespaceType:

    def __init__(self, options: NamespaceTypeOptions, nestedNamespaces: ['NamespaceType']):
        self.options = options
        self.nestedNamespaces = nestedNamespaces
